tests/integration/flow/test_crud.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
the integration test should talk about some feature, the `flow` name is too generic, can you call it `crud`?

Reply:
okay